fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/FenixDebugDrawerDestinations.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.navigation
+
+import org.mozilla.fenix.R
+
+/**
+ * The navigation route for [DebugDrawerLandingScreen].
+ */
+const val DEBUG_DRAWER_ROUTE_LANDING_SCREEN = "landing_screen"
```

Comment:
is the 'landing screen' verbiage already used? I wonder if something like 'main menu' or 'drawer home' would be more consistent with the current naming?

Reply:
It's not set in stone; I do have [another PR using it](https://github.com/mozilla-mobile/firefox-android/pull/5012), but I definitely wanted to get yalls thoughts on what to name this screen. The name though will only be visible in the code (nothing user facing) so:
- The composable name
- Route 
- Random variable naming

Reply:
so maybe:
- DebugDrawerHome for the composable
- "home" for the route

Reply:
>  DebugDrawerHome for the composable
    "home" for the route
    
Yeah I think this is good. I'm flip-flopping from 'home' to 'main' to 'parent' but don't have a strong preference for either